docs/whats-new/release-notes/v2_15_0.md

```diff
@@ -0,0 +1,150 @@
+
+- Simplified the preparation for JCL system tests.
+- Updated the version of zos-node-accessor to 1.0.16.
+
+### Zowe Explorer
+
+#### Zowe Explorer (Core)
+
+- See the [Zowe Explorer](https://github.com/zowe/vscode-extension-for-zowe/blob/main/packages/zowe-explorer/CHANGELOG.md) changelog for updates included in this release.
+
+#### Zowe Explorer API
+
+- See the [Zowe Explorer API](https://github.com/zowe/vscode-extension-for-zowe/blob/main/packages/zowe-explorer-api/CHANGELOG.md) changelog for updates included in this release.
+
+#### Zowe Explorer FTP Extension
+
+- See the [Zowe Explorer FTP Extension](https://github.com/zowe/vscode-extension-for-zowe/blob/main/packages/zowe-explorer-ftp-extension/CHANGELOG.md) changelog for updates included in this release.
+
+#### Zowe Explorer ESLint Plug-in
+
+- See the [Zowe Explorer ESLint Plug-in](https://github.com/zowe/vscode-extension-for-zowe/blob/main/packages/eslint-plugin-zowe-explorer/CHANGELOG.md) changelog for updates included in this release.
+
+## Bug fixes
+
+Zowe Version 2.15.0 contains the bug fixes that are described in the following topics.
+
+### Zowe installation and packaging
+ 
+### Zowe Application Framework
+
+#### Zlux Server Framework
+
+### Zowe API Mediation Layer
+
+ -Displaying the base path in API Catalog has been fixed. ([#3297](https://github.com/zowe/api-layer/issues/3297))
+- Disabling EhCache using `apiml.caching.enabled=false` has been fixed. ([#3280](https://github.com/zowe/api-layer/issues/3280))
+
+### Zowe CLI
+
+#### Zowe CLI (Core)
+
+- Fixed race condition in `config convert-profiles` command that may fail to delete secure values for old profiles.
+
+- Resolved technical currency by updating `socks` transitive dependency.
+
+- Updated zos-files copy dsclp system tests to include large mock files.
+
+#### Zowe CLI Imperative Framework
+
+- Fixed race condition in `config convert-profiles` command that may fail to delete secure values for old profiles.
+
+- Resolved issue in `ProfileInfo` where schema comparisons fail, specifically when comparing the cached schema against a command-based schema during registration.
+
+- Resolved technical currency by updating `socks` transitive dependency.
+
+- Updated `mustache` and `jsonschema` dependencies for technical currency.
+
+- Fixed issue where secure property names could be returned for the wrong profile. ([Zowe Explorer #2633](https://github.com/zowe/vscode-extension-for-zowe/issues/2633))
+
+- Fixed issue when a property is not found in `ProfileInfo.updateProperty({forceUpdate: true})`. ([Zowe Explorer #2493](https://github.com/zowe/vscode-extension-for-zowe/issues/2493))
+
+#### CICS Plug-in for Zowe CLI
+
+#### DB2 Plug-in for Zowe CLI
+
+#### MQ Plug-in for Zowe CLI
+
+#### IMS Plug-in for Zowe CLI
+
+ Added missing shrinkwrap
+
+#### z/OS FTP Plug-in for Zowe CLI
+
+- Fixed bug where upload dataset using Buffer, stead of string. ([2533](https://github.com/zowe/vscode-extension-for-zowe/issues/2533))
```

Comment:
@adam-wolfe - I don't understand this note at all. Can you help explain what we did, and why we did it?

Reply:
"stead" should be "instead". zos-node-accessor's uploadDataset function expects data in the form of a ReadableStream, Buffer, or path to a local file. 

Since the FTP plugin was passing a string rather than a Buffer, on Windows I believe there was an issue with JCL statements which contain "//" at the start of a line being misinterpreted as a local file path, causing the upload function to fail.

By uploading the data as a buffer, it resolves an issue with uploading JCL when using the plug-in on a computer running Windows.

@zFernand0 might have a more succinct way to explain it.

Reply:
That explanation is perfect! 🙏🏽 

I think we can say something along these lines:
- Fixed a Windows specific issue where uploading a dataset with JCL content hangs preventing further actions through the zFTP VSCode Extension.